release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -298,6 +298,94 @@ With this release, the `MaxUnavailableStatefulSet` featureset configuration para
 [id="ocp-4-14-monitoring"]
 === Monitoring
 
+The monitoring stack for this release includes the following new and modified features.
+
+[id="ocp-4-14-monitoring-updates-to-monitoring-stack-components-and-dependencies"]
+==== Updates to monitoring stack components and dependencies
+This release includes the following version updates for monitoring stack components and dependencies:
+
+* kube-state-metrics to 2.9.2
+* node-exporter to 1.6.1
+* prom-label-proxy to 0.7.0
+* Prometheus to 2.46.0
+* prometheus-operator to 0.67.1
+
+[id="ocp-4-14-monitoring-changes-to-alerting-rules"]
+==== Changes to alerting rules
+
+[NOTE]
+====
+Red{nbsp}Hat does not guarantee backward compatibility for recording rules or alerting rules.
+====
+
+* *New*
+** Added the `KubeDeploymentRolloutStuck` alert to monitor if the rollout of a deployment has not progressed for 15 minutes.
+** Added the `NodeSystemSaturation` alert to monitor resource saturation on a node.
+** Added the `NodeSystemdServiceFailed` alert to monitor the systemd service on a node.
+** Added the `NodeMemoryMajorPagesFaults` alert to monitor major page faults on a node.
+** Added the `PrometheusSDRefreshFailure` alert to monitor failed Prometheus service discoveries.
+
+* *Changed*
+** Modified the `KubeAggregatedAPIDown` alert and the `KubeAggregatedAPIErrors` alert to evaluate only metrics from the `apiserver` job.
+** Modified the `KubeCPUOvercommit` alert to evaluate only metrics from the `kube-state-metrics` job.
+** Modified the `NodeHighNumberConntrackEntriesUsed`, `NodeNetworkReceiveErrs` and `NodeNetworkTransmitErrs` alerts to evaluate only metrics from the `node-exporter` job.
+
+* *Removed*
+** Removed the `MultipleContainersOOMKilled` alert and replaced it with the `ContainersOOMKilled` alert, which uses the `container_runtime_crio_containers_oom_total` metric and notifies users if more than five containers on the host have been terminated by the out-of-memory (OOM) killer on the host.
+
+[id="ocp-4-14-monitoring-new-option-to-create-alerts-based-on-core-platform-metrics"]
+==== New option to create alerts based on core platform metrics
+
+With this release, administrators can create new alerting rules based on core platform metrics.
+You can now modify settings for existing platform alerting rules by adjusting thresholds and by changing labels.
+You can also define and add new custom alerting rules by constructing a query expression based on core platform metrics in the `openshift-monitoring` namespace.
+This feature was included as a technology preview in a previous {product-title} release and is now generally available.
```

Comment:
Is it worth mentioning which version of OCP the tech preview was in? Its not necessary but I was wondering your thoughts